awscrt/s3.py

```diff
@@ -180,6 +180,11 @@ class S3Client(NativeResource):
             Gigabits per second (Gbps) that we are trying to reach.
             You can also use `get_recommended_throughput_target_gbps()` to get recommended value for your system.
             10.0 Gbps by default (may change in future)
+
+        memory_limit (Optional[int]): Memory limit, in bytes, of how much memory
+            client can use for buffering data for requests.
+            Default values scale with target throughput and are currently
+            between 2GBs and 8GBs (might change in future.)
```

Comment:
trivial: consistent with `throughput_target_gbps` docs wording.
```suggestion
            between 2GBs and 8GBs (may change in future)
```